docs/gettingstarted.rst

```diff
@@ -0,0 +1,179 @@
+=================
+ Getting started
+=================
+
+This tutorial introduces you to scikit-fem.
+It assumes that you are familiar with the basics of the finite element method
+and how the method relates to partial differential equations.
+
+Step 0: Install scikit-fem
+==========================
+
+If you have a supported Python installation on your computer, you can run
+
+.. code-block:: bash
+
+   pip install scikit-fem
+
+If you do not want to install anything locally, you can try `Google Colab
+<https://colab.research.google.com/>`_ in your web browser and install scikit-fem
+by executing
+
+.. code-block:: bash
+
+   !pip install scikit-fem
+
+Step 1: Clarify the problem
+===========================
+
+In this tutorial we solve the system
+
+.. math::
+   \begin{aligned}
+        -\Delta u &= f \quad && \text{in $\Omega$,} \\
+        u &= 0 \quad && \text{on $\partial \Omega$,}
+   \end{aligned}
+
+where :math:`\Omega = [0, 1]^2` is a square domain
+and :math:`f(x,y)=\sin \pi x \sin \pi y`.
+The corresponding weak formulation reads:
+find :math:`u \in V` satisfying
+
+.. math::
+   \int_\Omega \nabla u \cdot \nabla v \,\mathrm{d}x = \int_\Omega fv\,\mathrm{d}x \quad \forall v \in V,
+
+where :math:`V = H^1_0(\Omega)`.
+
+Step 2: Express the forms as code
+=================================
+
+Next step is to write the forms
+
+.. math::
+
+   a(u, v) = \int_\Omega \nabla u \cdot \nabla v \,\mathrm{d}x \quad \text{and} \quad L(v) = \int_\Omega f v \,\mathrm{d}x
+
+as source code.  We write each form as a function and
+decorate it accordingly:
+
+.. doctest::
+
+   >>> import skfem as fem
```

Comment:
That's a nice idiom.  I like it.  It stops the linters complaining about importing *.